tests/data/SPECS/dynamic-fail.spec

```diff
@@ -0,0 +1,37 @@
+Summary: dynamic hello -- hello, world rpm
+Name: dynamic
+Version: 1.0
+Release: 1
+Group: Utilities
+License: GPL
+Distribution: RPM test suite.
+URL: http://rpm.org
+BuildArch:	noarch
+
+%description
+Simple rpm demonstration.
+
+%prep
+%setup -q -T -c
+
+%build
+echo "Q: Why?\nA: Because we can!" > FAQ
+
+%install
+mkdir -p $RPM_BUILD_ROOT/usr/local/bin
+echo " " > $RPM_BUILD_ROOT/usr/local/bin/hello
+
+
+echo "%package docs" >> %{specpartsdir}/docs.specpart
+echo "Summary: Documentation for dynamic spec" >> %{specpartsdir}/docs.specpart
+echo "BuildArch: noarch" >> %{specpartsdir}/docs.specpart
+echo "%files docs" >> %{specpartsdir}/docs.specpart
```

Comment:
This fail case could be trivially handled with the same spec as the success case, just add a macro that swaps the behavior as needed. That's how most of the test-suite is done to avoid adding dozens of specs only differing by a line or two.

Reply:
Done. I guess I started with a fool prove variant to not debug the test case and the error handling at once.